tortoise/verifying.go

```diff
@@ -155,7 +157,7 @@ func (v *verifying) verify(logger log.Log, lid types.LayerID) bool {
 		logger,
 		layer.blocks,
 		func(block *blockInfo) sign {
-			if block.height > layer.verifying.referenceHeight {
+			if block.height > refheight {
 				return neutral
```

Comment:
i keep forgetting why it votes neutral instead of against here. can you remind me please, or maybe add comment?